### DISCUSSION TITLE
How can I download the new version in Ubuntu 18.04 Bionic

### DISCUSSION THREAD

**Discussion by ghost** [abraunegg/onedrive#1369] [Q&A] [answered]
[Disclaimer: Noob]

I have many many problems with installing. First time the package got downloaded from ubuntu repository, because somehow adding the yannic repo via terminal did not work (Ubuntu). I tried adding it to the sources.list but still adding repo still reads this:

```
Ign:1 http://download.opensuse.org/repositories/home:/ungoogled_chromium/Ubuntu_Bionic  InRelease
Ign:2 http://ppa.launchpad.net/yann1ck/onedrive/ubuntu bionic InRelease        
Err:3 http://download.opensuse.org/repositories/home:/ungoogled_chromium/Ubuntu_Bionic  Release
  404  Not Found [IP: 195.135.221.134 80]
Get:4 https://deb.debian.org/debian testing InRelease [142 kB]                 
Hit:5 http://ppa.launchpad.net/sebastian-stenzel/cryptomator/ubuntu bionic InRelease
Hit:6 http://security.ubuntu.com/ubuntu bionic-security InRelease              
Hit:7 http://ppa.launchpad.net/wereturtle/ppa/ubuntu bionic InRelease          
Err:4 https://deb.debian.org/debian testing InRelease                          
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138
Hit:8 http://ppa.launchpad.net/zorinos/apps/ubuntu bionic InRelease            
Hit:9 http://ppa.launchpad.net/zorinos/drivers/ubuntu bionic InRelease         
Hit:10 http://ppa.launchpad.net/zorinos/patches/ubuntu bionic InRelease        
Hit:11 https://packages.expandrive.com/expandrive stable InRelease             
Hit:12 http://ppa.launchpad.net/zorinos/stable/ubuntu bionic InRelease         
Err:13 http://ppa.launchpad.net/yann1ck/onedrive/ubuntu bionic Release         
  404  Not Found [IP: 91.189.95.85 80]
Hit:14 https://packages.zorinos.com/stable bionic InRelease                    
Hit:15 https://packages.zorinos.com/patches bionic InRelease                   
Hit:16 https://packages.zorinos.com/apps bionic InRelease                      
Hit:17 https://packages.zorinos.com/drivers bionic InRelease                   
Err:18 http://hu.archive.ubuntu.com/ubuntu bionic InRelease                    
  Cannot initiate the connection to hu.archive.ubuntu.com:80 (2a09:7ac0::4). - connect (101: Network is unreachable) Could not connect to hu.archive.ubuntu.com:80 (45.67.159.244), connection timed out
Err:19 http://hu.archive.ubuntu.com/ubuntu bionic-updates InRelease
  Cannot initiate the connection to hu.archive.ubuntu.com:80 (2a09:7ac0::4). - connect (101: Network is unreachable)
Err:20 http://hu.archive.ubuntu.com/ubuntu bionic-backports InRelease
  Cannot initiate the connection to hu.archive.ubuntu.com:80 (2a09:7ac0::4). - connect (101: Network is unreachable)
Reading package lists... Done
```

How can I install the latest package? Whats the problem?

**Comment by abraunegg** [comment, marked as answer]
@Hogerys 
The PPA does not support Ubuntu Bionic (18.x). Refer to https://launchpad.net/~yann1ck/+archive/ubuntu/onedrive for supported platforms.

Your only options are:
* Upgrade your Ubuntu platform to a Ubuntu version that is supported by the PPA
* Build from source following the instructions for Ubuntu 18.x

If you wish to change this situation so that you can just use 'sudo apt install onedrive' from any Ubuntu platform without the need for a PPA, consider becoming the Ubuntu package maintainer and contribute back to the community.

**Reply by ghost** [reply]
I wish i knew how to contribute. Ill try compiling. Hopefully i wont brick anything.

**Reply by ghost** [reply]
Is there a tutorial or a guide? I installed dependencies, ./configure, make but i have no idea whats next.

**Reply by ghost** [reply]
Oh i figured it out. Its basically installed. Ty for the help.

**Reply by abraunegg** [reply]
Make sure you 'uninstall' the onedrive package as well - it is broken, dont use the package. If you do not do this step, you could have 2 x application versions on your system which will cause further issues.

Additionally, if you are now building from source you also now need to rebuild your client monthly to ensure you get the latest release + bug fixes.
